src/pudl/transform/ferc714.py

```diff
@@ -343,7 +343,8 @@ def _standardize_offset_codes(df, offset_fixes):
     code = df["utc_offset_code"].mask(is_blank)
     # Apply specific fixes on a per-respondent basis:
     return code.groupby(df['respondent_id_ferc714']).apply(
-        lambda x: x.replace(offset_fixes[x.name]) if x.name in offset_fixes else x
+        lambda x: x.replace(
+            offset_fixes[x.name]) if x.name in offset_fixes else x
```

Comment:
Can you check and make sure that your editor is wrapping lines at 88 characters, and not at 79? I've seen a few of these auto-formatting changes come up.